docs/user-guide/content-sources.rst

```diff
@@ -98,6 +103,7 @@ The pulp-admin client can be use to list all defined content sources as follows:
   Base URL:       http://content-world/content/
   Enabled:        1
   Expires:        3d
+  Headers:        X-RHUI-ID=1234 \ X-Csrf-Token=31415926535897
```

Comment:
This is the way pulp-admin formats the content source when `pulp-admin content sources list` is called. 
Let me know if you want me to update `pulp-admin content sources list` formatting to show this on separate lines. 
But as far as the documentation goes I think it should match the returned format